README.md

```diff
@@ -167,3 +167,18 @@ effect. Once it's done, exec into the Jenkins pod and check the output of `ps
 command as space-separated and not comma-separated. Refer [this
 diff](https://github.com/openshift/openshift-docs/pull/7259/files?short_path=05f80f3#diff-05f80f3ab954ce57c630417065819109)
 to ensure that values are passed properly.
+
+Now lets setup weekly scan triggering mechanism. Note the pipelines will be created by seed job, but without triggers.
+
+First, setup a configmap that contains URL of openshift master as below. Note, ports if any, must be included.
```

Comment:
I keep asking for these changes -
The readme has same word written differently at other places, we should stick to unify represting all these product's names.

Please update the line
*ConfigMap
*OpenShift

Reply:
Nope. I see still see openshift and configmap.